app/Jobs/ProcessMediaWikiJobsJob.php

```diff
@@ -20,7 +21,7 @@ class ProcessMediaWikiJobsJob implements ShouldQueue, ShouldBeUnique
     public function __construct (string $wikiDomain)
     {
         $this->wikiDomain = $wikiDomain;
-        $this->jobsKubernetesNamespace = env('API_JOB_NAMESPACE', 'api-jobs');
+        $this->jobsKubernetesNamespace = Config::get('wbstack.api_job_namespace');
```

Comment:
This is a drive by fix for a mistake made earlier on this year. It will works as before, but we probably want to update the chart at some point as well to use `WBSTACK_` env vars throughout.